src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/util/LicenceStatus.kt

```diff
@@ -30,5 +30,28 @@ enum class LicenceStatus {
         RECALLED -> LicenceEventType.RECALLED
       }
     }
+
+    fun searchRelevantLicenceStatus(): List<LicenceStatus> {
+      return listOf(
+        ACTIVE,
+        IN_PROGRESS,
+        SUBMITTED,
+        APPROVED,
+        VARIATION_IN_PROGRESS,
+        VARIATION_SUBMITTED,
+        VARIATION_APPROVED,
+        VARIATION_REJECTED,
+      )
```

Comment:
We could raised the layer of abstraction here rather than name based on how they're used. 

I think this is the same list that we've named "inflight licences" in other parts of the code base, maybe that would be a good name for them here?

Reply:
Done